### PR TITLE
[zh] Resync task files (5)

### PR DESCRIPTION
**Pull request by tengqm**
Related: #27674

### CODE REVIEW

File: content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 1]
```diff
@@ -154,20 +152,23 @@ You now have to ensure that the kubectl completion script gets sourced in all yo
...
 <!-- 
-- If you installed kubectl with Homebrew (as explained [above](#install-with-homebrew-on-macos)), then the kubectl completion script should already be in `/usr/local/etc/bash_completion.d/kubectl`. In that case, you don't need to do anything.
+- If you installed kubectl with Homebrew (as explained [here](/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)), then the kubectl completion script should already be in `/usr/local/etc/bash_completion.d/kubectl`. In that case, you don't need to do anything.
 -->
-- 如果你是用 Homebrew 安装的 kubectl（[如上所述](#install-with-homebrew-on-macos)），
-  那么 kubectl 补全脚本应该已经安装到目录 `/usr/local/etc/bash_completion.d/kubectl` 中了。
-  这种情况下，你什么都不需要做。
+- 如果你是用 Homebrew 安装的 kubectl（如
+  [此页面](/zh/docs/tasks/install-with-homebrew-on-macos/#install-with-homebrew-on-macos)
+  所描述），则kubectl 补全脚本应该已经安装到目录 `/usr/local/etc/bash_completion.d/kubectl`
```
Comment (howieyuen) [inline content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:159]: 则 kubectl

File: content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 2]
```diff
@@ -154,20 +152,23 @@ You now have to ensure that the kubectl completion script gets sourced in all yo
...
-   <!-- 
-   The Homebrew installation of bash-completion v2 sources all the files in the `BASH_COMPLETION_COMPAT_DIR` directory, that's why the latter two methods work.
-   -->
-   用 Hommbrew 安装的 bash-completion v2 会初始化 目录 `BASH_COMPLETION_COMPAT_DIR` 中的所有文件，这就是后两种方法能正常工作的原因。
-   {{< /note >}}
+  {{< note >}}
+  <!-- 
+  The Homebrew installation of bash-completion v2 sources all the files in the `BASH_COMPLETION_COMPAT_DIR` directory, that's why the latter two methods work.
+  -->
+  用 Hommbrew 安装的 bash-completion v2 会初始化 目录 `BASH_COMPLETION_COMPAT_DIR`
```
Comment (howieyuen) [inline content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:166]: 会初始化目录

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Review by chenrui333 — APPROVED** [review]
/lgtm
/approve

File: content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:159]: I found the code style issues here and there, maybe we can do batch updates in a later time. Let's just focus on the cleanup work right now :)

**Pull request merged**